samples/nrf9160/serial_lte_modem/src/main.c

```diff
@@ -26,7 +26,8 @@ static struct k_work exit_idle_work;
 
 /* global variable used across different files */
 struct at_param_list m_param_list;
-struct modem_param_info modem_param;
+struct modem_param_info m_modem_param;
+char m_buf[128];
 
```

Comment:
What about defining a buffer length configuration instead of hard coding to 128?

Reply:
I'll use AT_CMD module defined length (not used so far as it's too long)